public/app/features/dashboard/components/PanelEditor/PanelEditorTableView.tsx

```diff
@@ -30,9 +30,9 @@ export function PanelEditorTableView({ width, height, panel, dashboard }: Props)
   // Subscribe to panel event
   useEffect(() => {
     const timeSrv = getTimeSrv();
-    const timeData = applyPanelTimeOverrides(panel, timeSrv.timeRange());
-
+    let timeData = applyPanelTimeOverrides(panel, timeSrv.timeRange());
     const sub = panel.events.subscribe(RefreshEvent, () => {
+      timeData = applyPanelTimeOverrides(panel, timeSrv.timeRange());
       panel.runAllPanelQueries(dashboard.id, dashboard.getTimezone(), timeData, width);
     });
```

Comment:
I think this change makes sense, but I suggest calculating the time data only in the subscription, as the first calculation is not used at all:

```suggestion
    const sub = panel.events.subscribe(RefreshEvent, () => {
      const timeData = applyPanelTimeOverrides(panel, timeSrv.timeRange());
      panel.runAllPanelQueries(dashboard.id, dashboard.getTimezone(), timeData, width);
    });
```